backend/src/controllers/chats.js

```diff
@@ -16,8 +15,8 @@ exports.list = async (ctx) => {
 exports.create = async (ctx) => {
   const params = ctx.request.body;
 
-  const chat = await database.Chat.create({message: params.message});
+  const chat = await database.Chat.create({ message: params.message });
```

Comment:
Do we need both chat and event in here?

Reply:
I think it's fine that chat remains as it was also in lesson3-nodejs-template